electrical/motor_controller.py

```diff
@@ -42,7 +44,7 @@ def go_forward(self):
             print('go_forward')
         else:
             GPIO.output([self.in1], GPIO.HIGH)
-            GPIO.output([self.in2], GPIO.HIGH)
+            GPIO.output([self.in2], GPIO.LOW)
```

Comment:
does this make it go forward or do a swing turn?

Reply:
See comment above